docs/guides/howto_manage_private_git.md

```diff
@@ -0,0 +1,37 @@
+# How to make QDT work with a private Git repository
+
+For now, QDT does not support private Git repositories. It's a "to be funded" issue. If you are willing to see this feature land in the functional perimeter, don't hesitet to get in touch!
+
+```{mermaid}
+---
+title: QDT workflow around a Git private forge (to be funded)
+---
+
+flowchart LR
+    A["QGIS/QDT admin"] <---> |"classic git workflow<br/>clone/pull & git push"| D(("Private Git repository<br/>i.e. git.myorg.com or on GitHub/Lab"))
+    D --->|"QDT"| F["QGIS end-user<br/>profiles"]
+```
+
+In the meanwhile, it's not a blocking lack, since QDT support local git repositories, i.e. stored on a network drive. Here comes a diagram illustrating the typical workflow used by most of end-users:
+
+```{mermaid}
+---
+title: QDT workflow around a Git private forge (workaround)
+---
+
+flowchart LR
+    A["QGIS/QDT admin"] <---> |"classic git workflow<br/>clone/pull & git push"| D(("Private Git repository<br/>i.e. git.myorg.com or on GitHub/Lab"))
+    D -->|"Pull<br/>(with deployment token)"| E[["Server on local network<br/>i.e. '//gis/software/qgis/qdt/profiles/'"]]
+    E --->|"QDT"| F["QGIS end-user<br/>profiles"]
+```
+
+To perform the intermediary git pull, it depends on the organization and GIS team habits. It can be performed manually through the command-line, with a GUI like the excellent [GitHub Desktop](https://github.com/apps/desktop) or with a script. We give below an example in PowerShell for Windows.
```

Comment:
```suggestion
To execute the intermediate git pull, the approach depends on the organization and GIS team habits. This can be done manually through the command-line, with a GUI like the excellent [GitHub Desktop](https://github.com/apps/desktop) or with a script. Below is an example PowerShell script for Windows.
```